### DISCUSSION TITLE
Reading polls with Tweepy

### DISCUSSION THREAD

**Discussion by a2435191** [tweepy/tweepy#1437] [Questions] [answered]
I've done my research, and I know it's impossible to add polls as a status with Tweepy. Is it possible to *read* polls, though? On [this article](https://developer.twitter.com/en/docs/twitter-api/v1/enrichments/overview/poll-metadata), it says that poll metadata is a free enrichment... does that mean I can get the data through Tweepy (or even requests) without paying?

**Comment by Harmon758** [comment, marked as answer]
[Poll metadata is only available for Enterprise APIs in v1.1 of Twitter's API](https://developer.twitter.com/en/docs/twitter-api/v1/data-dictionary/object-model/entities#polls), which is what the documentation you linked is for.

[Poll objects](https://developer.twitter.com/en/docs/twitter-api/data-dictionary/object-model/poll) are available with Twitter API v2:
https://twittercommunity.com/t/announcing-early-access-to-the-next-generation-of-the-twitter-api/139612#poll-results
https://developer.twitter.com/en/docs/twitter-api/data-dictionary/introduction
https://developer.twitter.com/en/docs/twitter-api/early-access
https://developer.twitter.com/en/docs/twitter-api/migrate
https://blog.twitter.com/developer/en_us/topics/tools/2020/introducing_new_twitter_api.html

For support for Twitter API v2 in Tweepy, see #1472.
